extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -934,30 +946,31 @@ class CornerstoneViewportService extends PubSubService implements IViewportServi
     const displaySetInstanceUIDs = this.viewportsDisplaySets.get(viewport.id);
 
     // Find overlay display sets (e.g. SEG, RTSTRUCT)
-    const overlayDisplaySet = displaySetInstanceUIDs
+    const overlayDisplaySets = displaySetInstanceUIDs
```

Comment:
Prior to this we were only ever processing ONE overlay display set. But now we do them ALL.